### PR TITLE
Fix relative path -c option CLI

### LINKED ISSUE #3115: CLI option -c requires relative path
### Expected behavior
`-c` should accept a path starting from the root of Lisk Core.

### Actual behavior
`-c ../../../../config/default/config.json`
Starts from chain module for some reason.

### Steps to reproduce
Run `npm start -c path`

### Which version(s) does this affect? (Environment, OS, etc...)
1.6.0

### PR DESCRIPTION
**Pull request by michielmulders**
### What was the problem?
The path that needed to be passed to `-c` option for starting Lisk Core from CLI required relative path like this `../../../../config/custom/config.json`.

### How did I fix it?
Fix the path to start from root.
You can use `-c` like this now `npm start -- -c config/cust/config.json`
### Review checklist

* The PR resolves #3115 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/modules/chain/helpers/config.js [hunk 1]
```diff
@@ -104,7 +104,12 @@ function Config(packageJson, parseCommandLineOptions = true) {
 
 	let customConfig = {};
 	if (program.config || process.env.LISK_CONFIG_FILE) {
-		customConfig = loadJSONFile(program.config || process.env.LISK_CONFIG_FILE);
+		const customConfigPath = path.join(
+			__dirname,
+			'../../../../../',
```
Comment (fchavant) [inline framework/src/modules/chain/helpers/config.js]: This looks more like a workaround more than a fix of the actual issue.

File: framework/src/modules/chain/helpers/config.js [hunk 2]
```diff
@@ -104,7 +104,12 @@ function Config(packageJson, parseCommandLineOptions = true) {
...
+			program.config || process.env.LISK_CONFIG_FILE
```
Comment (fchavant) [inline framework/src/modules/chain/helpers/config.js]: What happens if I pass an absolute path to `-c`?

Comment (michielmulders) [reply framework/src/modules/chain/helpers/config.js]: Do we want that behavior? Until now, the path was relative from root of project. @fchavant

Comment (fchavant) [reply framework/src/modules/chain/helpers/config.js]: OK that's fine then.

Comment (michielmulders) [reply framework/src/modules/chain/helpers/config.js]: Decided to keep the relative path for now.

File: framework/src/modules/chain/helpers/config.js (hunk 1 above)
Comment (michielmulders) [reply framework/src/modules/chain/helpers/config.js]: Patch it for now.
Other paths in folder are constructed in the same way.

**Comment by fchavant** [comment]
Noticed a bug while testing this PR: https://github.com/LiskHQ/lisk/issues/3123

**Pull request merged**